Metrics beta change disclaimer (#10283)
getsentry/sentry-docs: docs/product/metrics/index.mdx

<Include name="feature-stage-beta-metrics.mdx" />

> <Alert level="warning" title="Upcoming API Changes">
> We will be updating our API to ensure that metrics are explicitly connected to spans. This change will improve debugging capabilities, by ensuring that metrics are always contextually linked with all your Sentry data through traces. To continue using our product, you will need to migrate to the new APIs once released. Learn more.
> </Alert>

Reviewer: ```suggestion
<Alert level="warning" title="Upcoming API Changes">
We'll be updating our API to ensure that metrics are explicitly connected to spans. This change will improve debugging capabilities by ensuring that metrics are always contextually linked with all your Sentry data through traces. To continue using our product, you'll need to migrate to the new APIs once released. 
</Alert>
```
We can add the "Learn More"
